Mage.Sets/src/mage/cards/b/BlossomingTortoise.java

```diff
@@ -0,0 +1,134 @@
+package mage.cards.b;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.ActivatedAbility;
+import mage.abilities.common.EntersBattlefieldOrAttacksSourceTriggeredAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.MillCardsControllerEffect;
+import mage.abilities.effects.common.continuous.BoostControlledEffect;
+import mage.abilities.effects.common.cost.CostModificationEffectImpl;
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.TargetCard;
+import mage.target.common.TargetCardInYourGraveyard;
+import mage.util.CardUtil;
+
+import java.util.UUID;
+
+/**
+ *
+ * @author Susucr
+ */
+public final class BlossomingTortoise extends CardImpl {
+
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent("land creatures you control");
+
+    static {
+        filter.add(CardType.LAND.getPredicate());
+        filter.add(TargetController.YOU.getControllerPredicate());
```

Comment:
you don't need this predicate or "you control" in the message if you're using `BoostControlledEffect`